scanner.py

```diff
@@ -442,7 +447,23 @@ def start_touch(self, gcmd, touch_settings, verbose):
                     if not has_shown_retry_info:
                         gcmd.respond_info(f"Retry Attempt {int(retries)}")
                         has_shown_retry_info = True  # Set flag to True after showing the message
-
+                
+                if randomize > 0 and new_retry == 1:
+                        # Generate random offsets within ±5 units for both x and y
+                        x_offset = random.uniform(-randomize, randomize)
+                        y_offset = random.uniform(-randomize, randomize)
+                        
+                        cur_pos = self.toolhead.get_position()[:]
```

Comment:
Is there a reason for this line?

Reply:
> Is there a reason for this line?

Looks like its leftover code, I'll remove it